src/Stripe.net/Stripe.net.csproj

```diff
@@ -17,6 +17,7 @@
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <IncludeSymbols>true</IncludeSymbols>
+    <DebugType>embedded</DebugType>
```

Comment:
Does this get included in our release artifact/grow its size?

Reply:
Yes, adds 100kb to a 3.9mb package.